common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -9,24 +9,27 @@ trait CommercialSwitches {
     SwitchGroup.Commercial,
     "dfp-caching",
     "Have Admin will poll DFP to precache adserving data.",
+    owners = Seq(Owner.withName("commercial team")),
     safeState = On,
     sellByDate = never,
     exposeClientSide = false
   )
 
   val HeaderBiddingUS = Switch(
-     SwitchGroup.Commercial,
-     "header-bidding-us",
-     "Auction adverts on the client before calling DFP (US edition only)",
-     safeState = Off,
-     sellByDate = never,
-     exposeClientSide = true
+    SwitchGroup.Commercial,
+    "header-bidding-us",
+    "Auction adverts on the client before calling DFP (US edition only)",
+    owners = Seq(Owner.withGithub("regiskuckaertz ")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
I feel like high-level switches like that (disable commercial or discussion) should be handled by a circuit breaker (and they probably are already). If we want to completely turn off comments on the website, setting the ASG size to 0 should be enough?
That being said, maybe there are features that aren't mapped to a microservice and that would need a flag as you're suggesting.